docs/topics/basic-syntax.md

```diff
@@ -68,6 +68,84 @@ fun main() {
 ```
 {kotlin-runnable="true" kotlin-min-compiler-version="1.3" id="kotlin-basic-syntax-println"}
 
+## Read from the standard input
+
+Use the function `readln()` to read data from the standard input. It reads the whole line as a string:
+
+```kotlin
+fun main() {
+//sampleStart
+    // Reads and stores the user input in a variable. For example: Hi there!
+    val myInput = readln()
+
+    println(myInput)
+    // Hi there!
+
+    // Reads and prints the user input without storing it in a variable. For example: Hi, Kotlin!
+    println(readln())
+    // Hi, Kotlin!
+
+//sampleEnd
+}
+```
+
+For user interaction, you can use the `pintln()`, `readln()`, and `print()` functions together to print messages requesting 
+and showing user input:
+
+```kotlin
+fun main() {
```

Comment:
Are the samples intentionally not runnable?

Reply:
Yes because we don't support reading standard input in our runnable code samples. So is for users to see examples but not test them.